front/src/Phaser/Game/GameScene.ts

```diff
@@ -1740,6 +1746,25 @@ ${escapedMessage}
         }
     }
 
+    private tryMovePlayerWithMoveToUuidParameter(): void {
+        const uuidParam = urlManager.getHashParameter("moveToUuid");
```

Comment:
We could maybe rename the parameter "moveToUser" ? (as other things might have a uuid and here, we are definitely asking to go to a user?)